### PR TITLE
refactor: add exceptions to orphans script and remove remaining module

### PR DESCRIPTION
**Pull request by andrewHEguardian**
## What are you doing in this PR?

Add some more exceptions to the orphans script. Remove the last orphaned module.

[**Trello Card**](https://trello.com/c/tNRKjVly/1228-orphans-script-cleanup)

## How to test

Run `yarn orphans` - do not see any modules flagged!

## Screenshots
image

### CODE REVIEW

File: support-frontend/assets/components/svgs/feast.tsx [hunk 1]
```diff
@@ -1,48 +0,0 @@
-export default function Feast() {
```
Comment (andrewHEguardian) [inline support-frontend/assets/components/svgs/feast.tsx:1]: @paul-daniel-dempsey I assume this wasn't going to come back soon?

Comment (paul-daniel-dempsey) [reply support-frontend/assets/components/svgs/feast.tsx:1]: Reluctant to remove the spoons I guess! Should of removed earlier, thanks.

Comment (andrewHEguardian) [reply support-frontend/assets/components/svgs/feast.tsx:1]: 🥄 We'll always have access to the code here if we remember 👍

File: support-frontend/webpack.entryPoints.js [hunk 2]
```diff
@@ -33,6 +33,6 @@ module.exports = {
 		ausMomentMap: 'pages/aus-moment-map/ausMomentMap.tsx',
 	},
 	ssr: {
-		ssrPages: 'helpers/rendering/ssrPages.ts',
+		ssrPages: 'scripts/build-ssr-content.tsx',
```
Comment (paul-daniel-dempsey) [inline support-frontend/webpack.entryPoints.js:36]: Presume this replaces old ssrPages.ts with a version that can display either `NewPaperArchive` or `HoldingContent`?

**Review by paul-daniel-dempsey — APPROVED** [review]
Great, works a charm

File: support-frontend/webpack.entryPoints.js (hunk 2 above)
Comment (andrewHEguardian) [reply support-frontend/webpack.entryPoints.js:36]: Yep, not sure if it should be a webpack entrypoint. The only purpose it serves here is to be an exception for the orphans script

Comment (andrewHEguardian) [reply support-frontend/webpack.entryPoints.js]: We don't need this

**Pull request merged**